.gitignore

```diff
@@ -24,3 +24,4 @@ jspm_packages/
 .serverless/
 
 .DS_Store
+lib/
```

Comment:
Is that supposed to be ignored?

Reply:
Yep. `tsconfig` says that "outDir" is set to "lib". If you run `yarn build`, it should build/transpile the `.ts` files into `lib/`. Hence, why I added to .gitignore

Reply:
we need src/ not lib/ 😄 nice pickup